README.md

```diff
@@ -104,99 +108,134 @@ To upload reports in other formats, modify your workflow by adding  the `upload-
 ```
 
 ## Configuring Analysis with dotTEST
+
 You can configure analysis with Parasoft dotTEST in one of the following ways:
- - By customizing the `Run Parasoft dotTEST` action directly in your GitHub workflow. See [Action Parameters](#action-parameters) for a complete list of available parameters.
- - By configuring options in Parasoft dotTEST tool. We recommend creating a `dottestcli.properties` file that includes all the configuration options and adding the file to dotTEST's working directory  - typically, the root directory of your repository. This allows dotTEST to automatically read all the configuration options from that file. See [Parasoft dotTEST User Guide](https://docs.parasoft.com/display/DOTTEST20221) for details.
- 
+
+- By customizing the `Run Parasoft dotTEST` action directly in your GitHub workflow. See [Action Parameters](#action-parameters) for a complete list of available parameters.
+- By configuring options in Parasoft dotTEST tool. We recommend creating a `dottestcli.properties` file that includes all the configuration options and adding the file to dotTEST's working directory  - typically, the root directory of your repository. This allows dotTEST to automatically read all the configuration options from that file. See [Parasoft dotTEST User Guide](https://docs.parasoft.com/display/DOTTEST20222) for details.
```

Comment:
double space before - typically

Reply:
I think it is okay here, double space should be applied when these would be subitems of list items